src/components/message/message.tsx

```diff
@@ -217,6 +223,15 @@ const MessageContent = ({
           )}
         </Box>
       )}
+
+      {showCopyIcon && (
+        <Flex paddingY={"16px"} gap={2} alignItems={"center"} maxH={"max-content"} marginTop={"16px"}>
+             <Text fontSize="14px"  fontWeight={500}>
```

Comment:
can we make the text clickable too? so now when you click "copy", nothing happens. We should make the both the icon and text a clickable button